### PR TITLE
Content changes for the transfer agreement pages

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: app/views/partials/transferAgreementFormGroup.scala.html [hunk 1]
```diff
@@ -6,27 +6,35 @@
 
 @(transferAgreementFormData: Form[_],
   taFormNamesAndLabels: Seq[(String, String)],
-  formAlreadySubmitted: Boolean
+  formAlreadySubmitted: Boolean,
+  formHeading: String
```
Comment (TomJKing) [inline app/views/partials/transferAgreementFormGroup.scala.html:10]: Maybe change this param name to match what it is, eg: `fieldSetLegend: String`

File: app/controllers/TransferAgreementPart2Controller.scala [hunk 2]
```diff
@@ -55,17 +54,18 @@ class TransferAgreementPart2Controller @Inject() (
       reference <- consignmentService.getConsignmentRef(consignmentId, request.token.bearerAccessToken)
     } yield {
       val formAndLabels = taFormNamesAndLabels.filter(f => taForm.formats.keys.toList.contains(f._1))
+      val formHeading = "I confirm that the Departmental Records Officer (DRO) has signed off on the following:"
```
Comment (TomJKing) [inline app/controllers/TransferAgreementPart2Controller.scala:57]: See comment below about the naming of this param

File: app/controllers/TransferAgreementPart1Controller.scala [hunk 3]
```diff
@@ -54,6 +53,7 @@ class TransferAgreementPart1Controller @Inject() (
     } yield {
       val formAndLabel = transferAgreementFormNameAndLabel.filter(f => taForm.formats.keys.toList.contains(f._1))
       val warningMessage = Messages("transferAgreement.warning")
+      val formHeading = "I confirm that all records are:"
```
Comment (TomJKing) [inline app/controllers/TransferAgreementPart1Controller.scala:56]: See comment below about the naming of this param.

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. 

Also can you make sure this change doesn't break the e2e tests. If it does, can you you a PR ready for the e2e test changes before merging these changes

File: app/views/partials/transferAgreementFormGroup.scala.html (hunk 1 above)
Comment (vimleshtna) [reply app/views/partials/transferAgreementFormGroup.scala.html]: Changed `formHeading` to `fieldSetLegend` everywhere, thanks.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks. Please make sure the e2e tests work with these changes. If not have a PR ready for fixing the tests.

**Pull request merged**